src/NServiceBus.AcceptanceTests/Performance/MessageDurability/When_disabling_durable_messages_on_durable_transport.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.AcceptanceTests.Performance.MessageDurability
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_disabling_durable_messages_on_durable_transport : NServiceBusAcceptanceTest
```

Comment:
Should this be `When_disabling_durable_messages_on_durable_only_transport` to match the other new test?